models/user.js

```diff
@@ -10,10 +12,24 @@ var userSchema = mongoose.Schema({
         email: String,
         username: String,
         password: String
+    },
+    status: {
```

Comment:
I know this is still preliminary but I think we should have a discussion of whether or not we should further abstract "status". For example I don't think we need to return the lynch vote or last will of the player every single time we call their status - that could be in a separate endpoint for sure.